nrf-hal-common/Cargo.toml

```diff
@@ -30,6 +32,11 @@ version = "1.0.2"
 default-features = false
 version = "0.2.3"
 
+[dependencies.nrf51]
+optional = true
+version = "0.9.0"
+git = "https://github.com/chocol4te/nrf51"
```

Comment:
This needs to be changed before merge

Reply:
Fixed, now https://github.com/nrf-rs/nrf51/pull/9 should be reviewed and merged :)

Reply:
Published it! You'll need to push something since I can't request a rebuild from Travis.